test/src/commands/create/e2e/dart_package/dart_pkg_test.dart

```diff
@@ -19,43 +19,56 @@ void main() {
       );
       expect(result, equals(ExitCode.success.code));
 
-      final formatResult = await Process.run(
+      final workingDirectory = path.join(directory.path, 'very_good_dart');
+
+      // add coverage to collect coverage on dart test
```

Comment:
I don't think comments are necessary. In this case I think the code is expressive and it is clear to know what it does making the comment (under my perspective) redundant.

```suggestion
```

However, if you think they are useful I think they should be capitalised and include a full stop.
```suggestion
      // Add coverage to collect coverage on dart test.
```

The same applies for the other occurrences of this exact comment and the other comment that says:
```dart
\\ collect coverage
```